Easydict/NewApp/View/MenuView/MainMenuShortcutCommand.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MainMenuShortcutCommand.swift
+//  Easydict
+//
+//  Created by Sharker on 2024/2/4.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import SwiftUI
+
+extension EasyDictMainMenu {
+    struct MainMenuShortcutCommand: Commands {
+        @State private var appShortcutCommandList = [
+            MainMenuShortcutCommandDataItem(title: ShortcutType.clearInput.localizedStringKey(), type: .clearInput),
```

Comment:
<img width="581" alt="Screenshot 2024-02-07 at 09 20 00" src="https://github.com/tisfeng/Easydict/assets/103433299/df70e05f-f7b7-4c72-b3b7-d23ff1aef0f8">

The title parameter is not not required since you have a title key in this enum.